packages/e2e-tests/fixtures/blocks/core__pullquote__deprecated-4.parsed.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "blockName": "core/pullquote",
+        "attrs": {
+            "mainColor": "black",
+            "textColor": "white",
+            "className": "is-style-solid-color"
+        },
+        "innerBlocks": [],
+        "innerHTML": "\n<figure class=\"wp-block-pullquote has-background has-black-background-color is-style-solid-color\"><blockquote class=\"has-text-color has-white-color\"><p>pullquote</p><cite>before block supports</cite></blockquote></figure>\n",
+        "innerContent": [
+            "\n<figure class=\"wp-block-pullquote has-background has-black-background-color is-style-solid-color\"><blockquote class=\"has-text-color has-white-color\"><p>pullquote</p><cite>before block supports</cite></blockquote></figure>\n"
+        ]
+    },
+    {
+        "blockName": null,
+        "attrs": {},
+        "innerBlocks": [],
+        "innerHTML": "\n",
+        "innerContent": [
+            "\n"
+        ]
+    }
```

Comment:
This is a bit weird -- is the parser tripping over something? 🤔

Reply:
Thanks @ockham 

I _think_ I've seen this before. When a new line exists at the end of a html fixture file, it seems as if the parser interprets this as an empty block. 😱 

The result is an empty block object in the parsed JSON (`*.parsed.json` files).


```
   {
        "blockName": null,
        "attrs": {},
        "innerBlocks": [],
        "innerHTML": "\n",
        "innerContent": [
            "\n"
        ]
    }
```

Changing [this line](https://github.com/WordPress/gutenberg/blob/3da717b8d0ac7d7821fc6d0475695ccf3ae2829f/packages/e2e-tests/fixtures/utils.js#L61) to something like `file: fileContents ? fileContents.trim() : null,` might help?

I'll try it out later.

Reply:
I created a PR to remove them: https://github.com/WordPress/gutenberg/pull/30981

Reply:
From my experience the empty block object is caused by a blank line at the end of the fixture file. Thanks for putting together the PR to clean up all fixtures 👍 

I think you'll also need to update the new pullquote deprecation in this PR, removing the blank line from it. The original deprecations are caught in #30981 by the looks of it.